iphone/Classes/TiUIListView.m

```diff
@@ -945,6 +945,10 @@ - (void)cleanup:(id)unused
   if (_footerViewProxy != nil) {
     [_footerViewProxy windowWillClose];
   }
+
+  if ([searchController isActive]) {
+    [searchController setActive:NO];
```

Comment:
```suggestion
    searchController.active = NO;
```
Please use property style accessors whenever possible here and in the other places where you check if the search bar controller is active.